core/src/main/java/com/scalar/db/storage/dynamo/DynamoTableMetadataManager.java

```diff
@@ -214,16 +214,20 @@ public void addTableMetadata(String namespace, String table, TableMetadata metad
                     .map(pKey -> AttributeValue.builder().s(pKey).build())
                     .collect(Collectors.toList()))
             .build());
-    itemValues.put(
-        CLUSTERING_KEY,
-        AttributeValue.builder()
-            .l(
-                metadata.getClusteringKeyNames().stream()
-                    .map(pKey -> AttributeValue.builder().s(pKey).build())
-                    .collect(Collectors.toList()))
-            .build());
-    itemValues.put(
-        SECONDARY_INDEX, AttributeValue.builder().ss(metadata.getSecondaryIndexNames()).build());
+    if (!metadata.getClusteringKeyNames().isEmpty()) {
```

Comment:
Do we need this change in this PR?

Reply:
Yes, we need it. The empty field in the request of putting in dynamo will throw an exception so I fix it as well.